apis/kueue/v1beta1/workload_types.go

```diff
@@ -428,6 +428,14 @@ type WorkloadStatus struct {
 	// This field is reset after the Workload is evicted.
 	// +optional
 	ClusterName *string `json:"clusterName,omitempty"`
+
+	// nodesToReplace, if specified, holds the names of failed nodes running at least one pod of this workload.
+	// This field is for internal use, is set by the node failure controler and should not be set by the users.
+	// It is used to signal kueue scheduler to search for replacement of the failed nodes (if the native kube
+	// scheduler cannot do it automatically). Requires enabling the TASFaliedNodReplacement feature gate.
```

Comment:
```suggestion
	// nodesToReplace holds the names of failed nodes running at least one pod of this workload
	// when Topology-Aware Scheduling is used. This field should not be set by the users. 
	// It indicates Kueue's scheduler is searching for replacements of the failed nodes. 
	// Requires enabling the TASFaliedNodReplacement feature gate.
```
Mostly small tweaks (handle as proposals and feel free to rephrase).

However, " (if the native kube-scheduler cannot do it automatically)" - I think this sentence is a bit misleading. Kube-scheduler will not even try because NotReady nodes are excluded from scheduling. Sure, the condition
might be temporary, but the mechanism in Kueue is triggered regardless of kube-scheduler being able 
or not being able to find replacement.